enforcer-rules/src/site/apt/requireJavaVendor.apt.vm

```diff
@@ -30,8 +30,11 @@ Require Java Vendor
    The following parameters are supported by this rule:
    
    * message - an optional message to the user if the rule fails.
-   
-   * name - The name of the allowed JDK vendor.
+
+   * excludes - a list of banned JDK vendors.
+
+   * includes - a list of allowed JDK vendors to include. These are exceptions to the excludes.
```

Comment:
This is the opposite of how most Maven Plugins work: if includes is 1..5, these are the only allowed values. With excludes you can finetune the includes, e.g. excludes = 2,4. This would mean that the required value must be 1,3 or 5.
For consistency this should be changed.